frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/whitelisted-health-checks.ts

```diff
@@ -1,6 +1,4 @@
-// TODO(@rexagod): Add KCS/doc links once available.
-
 export const whitelistedHealthChecksRef = {
   MON_DISK_LOW:
-    'https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/troubleshooting_openshift_data_foundation/troubleshooting-alerts-and-errors-in-openshift-data-foundation_rhocs#resolving-cluster-health-issues_rhodf',
+    'https://dxp-docs.ext.us-west.aws.prod.paas.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html-single/troubleshooting_openshift_data_foundation/index#resolving-cluster-health-issues_rhodf',
```

Comment:
Why are we not doing access.redhat.com ?
The docs are live under that only ?

Reply:
Olive told me that the links based on `access.redhat.com` are broken currently. The redirection (for `access.redhat.com`) does not seem to be working at the moment, and the customer portal has been moved to this newer domain.

Reply:
we need to make sure that what we add here will be live after release of odf 4.9.
If its broken now then its okay, the url should be fixed for future.